rdkit/Chem/AllChem.py

```diff
@@ -76,9 +79,8 @@ def ComputeMolVolume(mol, confId=-1, gridSpacing=0.2, boxMargin=2.0):
   conf = mol.GetConformer(confId)
   CanonicalizeConformer(conf)
   box = ComputeConfBox(conf)
-  sideLen = ( box[1].x-box[0].x + 2*boxMargin, \
-              box[1].y-box[0].y + 2*boxMargin, \
-              box[1].z-box[0].z + 2*boxMargin )
+  sideLen = (box[1].x - box[0].x + 2 * boxMargin, box[1].y - box[0].y + 2 * boxMargin,
```

Comment:
I don't feel that strongly about it. The backslashes (though not needed) don't bother me, but I'd rather have the less-readable condensed form than give up yapf or add some kind of wart into the code to prevent the reformatting here.